bindings/python/gi/overrides/Modulemd.py

```diff
@@ -134,7 +136,28 @@ def get_xmd(self):
             variant_xmd = super().get_xmd()
             return variant_xmd.unpack()
 
-
     ModuleStreamV1 = override(ModuleStreamV1)
     __all__.append(ModuleStreamV1)
 
+    class ServiceLevel(Modulemd.ServiceLevel):
+
+        def set_eol(self, eol):
+            if (isinstance(eol, datetime.date)):
+                return super().set_eol_ymd(eol.year, eol.month, eol.day)
```

Comment:
Note that `super().foo` only works on py3. On py2, this should be `super(ServiceLeve, self).foo`.
The latter works with both though, so using that might be better.